indico/modules/rb_new/operations/conflicts.py

```diff
@@ -51,11 +52,12 @@ def get_rooms_conflicts(rooms, start_dt, end_dt, repeat_frequency, repeat_interv
     for room_id, occurrences in blocked_rooms.iteritems():
         rooms_conflicts[room_id] += get_room_blockings_conflicts(room_id, candidates, occurrences)
 
-    for room_id, occurrences in nonbookable_periods.iteritems():
-        rooms_conflicts[room_id] += get_room_nonbookable_periods_conflicts(candidates, occurrences)
+    if not rb_is_admin(session.user):
```

Comment:
can you add a TODO comment here as well about properly checking for room-specific override permissions?